test/kokkos_sparse_derivs_unit.K

```diff
@@ -2,7 +2,7 @@
 #include <iostream>
 #include <limits>
 
-#include "metaphysicl_config.h"
+#include "metaphysicl/metaphysicl_config.h"
```

Comment:
Why do we need the prefix here but not in any of the other includes of metaphysicl_config.h?

Reply:
That is a great question, mostly because I don't understand how it's working for the other tests
```
(moose-python) lindad@rod:/data/lindad/metaphysicl/build/src/utilities/include(kokkos)$ ls
metaphysicl
(moose-python) lindad@rod:/data/lindad/metaphysicl/build/src/utilities/include(kokkos)$ ls /data/lindad/metaphysicl/src/utilities/include/
metaphysicl
```